addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14292,7 +14295,7 @@ msgstr ""
 #. Description of setting "Music -> Library -> Clean library..." with label #334
 #: system/settings/settings.xml
 msgctxt "#36148"
-msgid "Remove items from your library that can't be found (either renamed, deleted, or on removable storage that is currently unplugged)."
+msgid "Remove items from your library that Can not be found (either renamed, deleted, or on removable storage that is currently unplugged)."
```

Comment:
Most existing entries are "can not"

I know cant't, cannot, can not are all correct, but why do we have to mix usage across?

No I dont mind which one we use here, but it should be consistent whatever that it.